File: StreamProviderWS/StreamProviderWS/Models/Common/MovieComment.cs
using System;

namespace StreamProviderWS.Models.Common
{
    public class MovieComment : Base
    {
        public string MovieId { get; set; }

        public User User { get; set; }

        public string Comment { get; set; }

        public double CurrentTime { get; set; } // seconds into the movie
    }
}

File: VideoStreamerUI/src/app/entities/movieComment.ts
import { SocialUser } from 'angularx-social-login';

export class MovieComment {

  Id: string;
  MovieId: string;
  User: SocialUser;

  Comment: string;
  CurrentTime: number;
  Shown: boolean;

  constructor(movieId: string, user: SocialUser, comment: string, currentTime: number) {
    this.MovieId = movieId;
    this.User = user;
    this.Comment = comment;
    this.CurrentTime = currentTime;
  }
}
